README.md

```diff
@@ -6,6 +6,82 @@ multibot
 
 A programmatic multi-repository helper.
 
+Multibot allows you to invoke actions (`read`, etc.) on 1+ files in 1+
+repositories concurrently. It is meant to be a helper tool for multi-project
+updates, branches, pull requests, etc.
+
+## Install
+
+```sh
+$ npm install multibot
+```
+
+## Usage
+
+```
+A friendly multi-repository robot.
+
+Usage: multibot --action=<string> [options]
+
+Options:
+  --action       Actions to take                               [string] [required] [choices: "read"]
+  --branch-src   Source git branch to read from                         [string] [default: "master"]
+  --branch-dst   Destination git branch to write to                                         [string]
+  --files        List of files (space delimited) to read / transform              [array] [required]
+  --org          GitHub organization for repos (can be instead specified on repos)          [string]
+  --repos        GitHub repositories (space delimited) of form `repo` or `org/repo`
+                                                                                  [array] [required]
+  --gh-user      GitHub user name (needs user + pass)                                       [string]
+  --gh-pass      GitHub password (needs user + pass)                                        [string]
+  --gh-token     GitHub token                                                               [string]
+  --transform    Path to transform JS file                                                  [string]
+  --format       Display output format  [string] [choices: "json", "text", "diff"] [default: "json"]
+  -h, --help     Show help                                                                 [boolean]
+  -v, --version  Show version number                                                       [boolean]
+
+Examples:
+  multibot --action=read --gh-token=TOKEN             Display the README file of multibot from
+  --org FormidableLabs --repos multibot --files       GitHub
+  README.md
+```
+
+## Actions
+
+Multibot can initiate various read-only and repository-mutating actions.
+
+### Transforms
```

Comment:
I think `## Transforms` should be its own second-level heading.

What I'm getting at is: I expect to see an enumaration of valid actions under the `## Actions` header. The first subheader that I see is `### Transforms`, and then `### read` below. It puts them in the same category, even though `transforms` isn't an action per se.